engaging-params.yaml

```diff
@@ -1,7 +1,8 @@
-readsdir: "/pool001/vanja1/nextflow/rawfastq/"
-outdir: "/pool001/vanja1/nextflow/output/"
-human_genome: "/pool001/vanja1/databases/kneaddata/"
-metaphlan_db: "/pool001/vanja1/databases/metaphlan/"
-humann_bowtie_db: "/pool001/vanja1/databases/humann/chocophlan"
-humann_protein_db: "/pool001/vanja1/databases/humann/uniref"
-humann_utility_db: "/pool001/vanja1/databases/humann/utility_mapping"
\ No newline at end of file
+readsdir: "test/single_end"
+outdir: "test/output/"
+human_genome: "/murray/databases/kneaddata/"
+metaphlan_db: "/murray/databases/metaphlan/"
+humann_bowtie_db: "/murray/databases/humann/chocophlan"
+humann_protein_db: "/murray/databases/humann/uniref"
+humann_utility_db: "/murray/databases/humann/utility_mapping"
+file_pattern: "{sample}.fastq.gz"
```

Comment:
```suggestion
filepattern: "*.fastq.gz"
```

Reply:
Actually, we should discard these changes like we talked about. `git reset origin/main engaging-params.yaml` should do it?